.github/workflows/deploy-swagger.yml

```
@@ -0,0 +1,48 @@
+
+on:
+  push:
+    branches: [ main ]
+
+jobs:
+  generate-swagger:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)